arm-sql/2015-05-01/swagger/sql.json

```diff
@@ -1,240 +1,157 @@
 {
   "swagger": "2.0",
   "info": {
+    "version": "2015-05-01-preview",
```

Comment:
I will do it today.

---
In reply to: [80462935](https://github.com/Azure/azure-rest-api-specs/pull/555#discussion_r80462935) [](ancestors = 80462935)